pkg/systemd/graphs.css

```diff
@@ -0,0 +1,147 @@
+@import "../lib/plot.css";
+
+.ct-graphs-layout {
+    --graph-padding: 1rem;
+    display: grid;
+    grid-template: repeat(4, 1fr) / 1fr;
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: var(--graph-padding);
+    grid-gap: var(--graph-padding);
+}
+
+.ct-graphs-layout > * {
+    overflow: hidden;
+}
+
+.server-graph-graph {
+    display: flex;
+}
+.server-graph-graph > * {
+    flex: auto;
+    display: flex;
+    width: 100% !important;
+    height: 100% !important;
+    max-height: 100%;
+    max-width: 100%;
+}
+
+.ct-graphs-layout .flot-base,
+.ct-graphs-layout .flot-text,
+.ct-graphs-layout .flot-overlay {
+    height: 100% !important;
+    width: 100% !important;
+    display: block;
+    flex: auto;
+}
+
+.server-graph-container {
+    max-height: 100%;
+    max-width: 100%;
+}
+
+@media (min-width: 641px) {
+    .ct-graphs-layout {
+        /* Set a max height so graphs don't get too wonky vertically */
+        max-height: 50rem;
+        grid-template: repeat(2, 1fr) / repeat(2, 1fr);
+    }
+}
+
+.server-graphs {
+    padding: 3rem;
+}
+
+/* Flot needs a little coaxing to behave */
+.server-graphs canvas,
+.server-graphs .flot-text {
+  max-width: 100%;
+  overflow: hidden;
+}
+
+#server-graph-toolbar {
+    margin-bottom: 1rem;
+}
+
+#server-graph-toolbar .ct-select,
+#server-graph-toolbar .dropdown {
+    display: inline-flex;
+}
+
+.ct-graph-simple {
+    display: grid;
+    grid-template-rows: max-content max-content 1fr;
+}
+
+.server-graph-container {
+    display: grid;
+    grid-template: "title" "graph" "legend";
+    grid-template-rows: max-content 1fr max-content;
+}
+
+.server-graph-title {
+    grid-area: title;
+}
+
+.server-graph-graph {
+    grid-area: graph;
+}
+
+.server-graph-legend {
+    grid-area: legend;
+    list-style-type: none;
+    text-align: center;
+    padding-top: 0.5rem;
+}
+
+.server-graph-legend li {
+    display: inline-block;
+}
+
+.server-graph-legend li + li {
+    margin-left: 1rem;
+}
+
+.server-graph-legend i {
+    font-size: 16px;
+    margin: 0;
+    padding: 0;
+}
+
+.server-graph-legend .cpu-io-wait i {
+    color: var(--color-red-1);
+}
+
+.server-graph-legend .cpu-kernel i {
+    color: var(--color-gold);
+}
+
+.server-graph-legend .cpu-user i {
+    color: var(--color-purple-6);
+}
+
+.server-graph-legend .cpu-nice i {
+    color: var(--color-green);
+}
+
+.server-graph-legend .memory-swap i {
+    color: var(--color-red-1);
+}
+
+.server-graph-legend .memory-cached i {
+    color: #c8eb79;
+}
+
+.server-graph-legend .memory-used i {
+    color: var(--color-blue);
+}
+
+.server-graph-legend .memory-free i {
+    color: var(--color-green);
+}
+
+/* FIXME: Why is this here on the page? Why is it blank? Hide for now */
+#server-graph-columns {
+    display: none !important;
```

Comment:
This is totally unclear indeed. The graphs page doesn't have a "hide" mode. Why wouldn't it just show the graphs by default directly? What happens if this gets dropped?

Reply:
Fixed.